grails-app/views/rolesManagement/index.gsp

```diff
@@ -0,0 +1,14 @@
+<html>
+    <head>
+        <meta name="layout" content="main_component">
+        <title>Roles Management</title>
+    </head>
+    <body>
+        <g:javascript>
+            const rolesConstants = {
+            };
+        </g:javascript>
+        <div id="rolesManagement"></div>
+        <asset:javascript src="build/rolesManagement.js"/>
+    </body>
+</html>
```

Comment:
Can we add a story (or update one that might exist) to cover converting this to react? This PR adds to the technical debt that we're trying to pay down in other stories.

Reply:
Here's a story for this request. 
[BTRX-712](https://broadinstitute.atlassian.net/browse/BTRX-712): ORSP - Convert Role Management

Reply:
Thank you!